pyprocessor/utils/config/config_manager.py

```
@@ -0,0 +1,2016 @@
+
+            # Add type
+            if "type" in value:
+                doc += f"**Type**: {value['type'].value}\n\n"
+
+            # Add default value
+            if "default" in value:
+                doc += f"**Default**: `{value['default']}`\n\n"
+
+            # Add required flag
+            if "required" in value and value["required"]:
+                doc += f"**Required**: Yes\n\n"
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)